### ISSUE TITLE
Crash SQLiteCompiledSql.java

### ISSUE THREAD

**Issue by duprass** [WalletConnect/WalletConnectKotlinV2#414]
**Describe the bug**

**SDK Version**
 - Client: Kotlin
 - Version: 2.0.0-rc.4

**To Reproduce**
Can't reproduce but received crash report.

**Expected behavior**
No crashes on initialization.

**Device (please complete the following information):**
 - Device: realme 3
 - OS: Android 10

**Additional context**
Crash log
```
net.sqlcipher.database.SQLiteCompiledSql.native_compile (SQLiteCompiledSql.java)
net.sqlcipher.database.SQLiteCompiledSql.compile (SQLiteCompiledSql.java:89)
net.sqlcipher.database.SQLiteCompiledSql.<init> (SQLiteCompiledSql.java:62)
net.sqlcipher.database.SQLiteProgram.<init> (SQLiteProgram.java:91)
net.sqlcipher.database.SQLiteQuery.<init> (SQLiteQuery.java:48)
net.sqlcipher.database.SQLiteDirectCursorDriver.query (SQLiteDirectCursorDriver.java:60)
net.sqlcipher.database.SQLiteDatabase.rawQueryWithFactory (SQLiteDatabase.java:2016)
net.sqlcipher.database.SQLiteDatabase.rawQuery (SQLiteDatabase.java:1902)
net.sqlcipher.database.SQLiteDatabase.keyDatabase (SQLiteDatabase.java:2669)
net.sqlcipher.database.SQLiteDatabase.openDatabaseInternal (SQLiteDatabase.java:2599)
net.sqlcipher.database.SQLiteDatabase.openDatabase (SQLiteDatabase.java:1247)
net.sqlcipher.database.SQLiteDatabase.openOrCreateDatabase (SQLiteDatabase.java:1322)
net.sqlcipher.database.SQLiteOpenHelper.getWritableDatabase (SQLiteOpenHelper.java:166)
net.sqlcipher.database.SupportHelper.getWritableDatabase (SupportHelper.java:83)
com.squareup.sqldelight.android.AndroidSqliteDriver$database$2.invoke (AndroidSqliteDriver.kt:34)
com.squareup.sqldelight.android.AndroidSqliteDriver$database$2.invoke (AndroidSqliteDriver.kt:33)
kotlin.SynchronizedLazyImpl.getValue (LazyJVM.kt:74)
com.squareup.sqldelight.android.AndroidSqliteDriver.getDatabase (AndroidSqliteDriver.kt:33)
com.squareup.sqldelight.android.AndroidSqliteDriver.access$getDatabase (AndroidSqliteDriver.kt:23)
com.squareup.sqldelight.android.AndroidSqliteDriver$executeQuery$1.invoke (AndroidSqliteDriver.kt:149)
com.squareup.sqldelight.android.AndroidSqliteDriver$executeQuery$1.invoke (AndroidSqliteDriver.kt:149)
com.squareup.sqldelight.android.AndroidSqliteDriver.execute (AndroidSqliteDriver.kt:123)
com.squareup.sqldelight.android.AndroidSqliteDriver.executeQuery (AndroidSqliteDriver.kt:149)
com.squareup.sqldelight.db.SqlDriver$DefaultImpls.executeQuery$default (SqlDriver.kt:35)
com.squareup.sqldelight.SimpleQuery.execute (Query.kt:69)
com.squareup.sqldelight.Query.executeAsList (Query.kt:134)
com.walletconnect.sign.storage.sequence.SequenceStorageRepository.getListOfSessionVOs (SequenceStorageRepository.kt:49)
com.walletconnect.sign.engine.domain.SignEngine.getListOfSettledSessions$sign_release (SignEngine.kt:422)
com.walletconnect.sign.client.SignProtocol$getListOfSettledSessions$1.invokeSuspend (SignProtocol.kt:260)
com.walletconnect.sign.client.SignProtocol$getListOfSettledSessions$1.invoke (Unknown Source:8)
com.walletconnect.sign.client.SignProtocol$getListOfSettledSessions$1.invoke (Unknown Source:2)
com.walletconnect.sign.client.SignProtocol$awaitLock$1.invokeSuspend (SignProtocol.kt:310)
kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith (ContinuationImpl.kt:33)
kotlinx.coroutines.DispatchedTask.run (DispatchedTask.kt:106)
```

**Comment by TalhaAli00** [comment]
we have merged in code that we think will fix this issue. This will be available with Sign 2.1.0

**Comment by TalhaAli00** [comment]
@duprass @soshial 
We have released new versions of our SDKs with a fix for this issue, you can find the complete list here https://github.com/WalletConnect/WalletConnectKotlinV2/tree/master
